content/en/docs/contribute/intermediate.md

```diff
@@ -513,6 +537,51 @@ most up-to-date version of that branch.
       the branch to your fork, and the pull request should no longer show any
       conflicts.
 
+12. You must squash multiple commits into a single commit before your PR can be
+merged. You can check the number of commits on your PR's `Commits` tab or by
```

Comment:
> You must squash multiple commits into a single commit before your PR can be merged.
For docs contributions, it's common not to insist on this. So I'm wary of “must”.

Reply:
Also, interactive rebase is a fairly advanced Git skill.

Reply:
@zacharysarah told me that squashing commits is a Kubernetes GitHub org policy, and Zach changed the wording to 'must'  in his earlier review of this PR. 

I agree that interactive rebase is advanced, which is why I recommend `git commit --amend` so contributors don't have to squash commits. The Community's _GitHub Workflow_ "Commit" [section]  (https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md#5-commit) also recommends `--amend`.